spack.yaml

```
@@ -0,0 +1,37 @@
+spack:
+  view: /opt/view
+  specs:
+  - icon ~aes ~jsbach ~ocean ~coupling ~rte-rrtmgp
+
+  'compilers:':
```

Why the quotes?

---

Don't remember if I did this on purpose. I think not
```suggestion
  compilers:
```